.github/workflows/merge-up.yml

```diff
@@ -27,4 +27,4 @@ jobs:
         uses: alcaeus/automatic-merge-up-action@main
         with:
           ref: ${{ github.ref_name }}
-          branchNamePattern: '<major>.<minor>'
+          branchNamePattern: '<major>.<minor>'
```

Comment:
The newline at end of file must be reverted.

Reply:
Ah this was unintentional, fixed